phpstan.neon.dist

```diff
@@ -0,0 +1,6 @@
+parameters:
+        level: 7
+        paths:
+        - "%currentWorkingDirectory%/lib/"
```

Comment:
Also need to cover tests: phpstan-phpunit may be needed

Reply:
added. Done